test/cql/cassandra_cql_test.result

```diff
@@ -755,12 +755,12 @@ OK
 > INSERT INTO lwt_on_static_columns_with_null_batch (a, b, d) values (2, 2, 'a')
 > UPDATE lwt_on_static_columns_with_null_batch  SET s = 2 WHERE a = 2 IF s = null
```

Comment:
Wow, this `s = NULL` is sad. Won't it return NULL unconditionally in SQL?

Reply:
(and `IS NULL` needs to be used to detect NULLs?)

Reply:
s = NULL would return a syntax error in SQL. But it depends on the dialect. IS NULL needs to be used to detect NULLs.